File: Average Salary Excluding the Minimum and Maximum Salary.cpp
class Solution {
public:
    double average(vector<int>& salary) {
        int i=1;
        int n=salary.size()-2;
        double sums=0;
        sort(salary.begin(),salary.end());
        while( i<=n){
        sums=sums+salary[i];
        ++i;
        }
        return sums/n;


    }
};

File: Average Salary Excluding the Minimum and Maximum Salary.py
class Solution:
    def average(self, salary: List[int]) -> float:
        i=1
        sums=0
        n=len(salary)-1
        salary.sort(reverse=True)  
        while i < n:
            sums=sums+salary[i]
            i=i+1
        return sums/(n-1)
